src/app/home/home.component.ts

```diff
@@ -134,6 +137,18 @@ export class HomeComponent {
   fixNumber(field: string) {
     this.options[field] = Number(this.options[field]);
   }
+  setInlineClass(enableInline: boolean) {
+    if (enableInline) {
+      this.toastr.overlayContainer = this.inlineContainers.first;
```

Comment:
If you toggle `enableInline` off and on after selecting 'B', the form still displays that 'B' is selected while 'A' is the component that is really is selected.